### ISSUE TITLE
git merge 和 rebase 的区别

### ISSUE THREAD

**Issue by muwenzi** [muwenzi/Blog#75]
**参考资料**
- [Git 分支 - 分支的衍合](https://git-scm.com/book/zh/v1/Git-%E5%88%86%E6%94%AF-%E5%88%86%E6%94%AF%E7%9A%84%E8%A1%8D%E5%90%88)
- [git rebase和merge的区别，rebase的运动过程和conflict的解决](http://www.tangshuang.net/2960.html)
- [Git Community Book 中文版: rebase](http://gitbook.liuhui998.com/4_2.html)

**Comment by muguan** [comment]
I suggest to involve the local and remote workflow with these two commands.

**Comment by muwenzi** [comment]
@muguan good idea, the main topic of this article is merge and rebase, but I will add the link about basic git commands and workflow, thanks.
